.github/workflows/build-android.yml

```diff
@@ -23,39 +23,54 @@ jobs:
     name: Build for Android
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
-        submodules: 'true'
-      continue-on-error: true
-    - name: Chocolatey
+        depth: 1
+        submodules: 'false'
+    - name: Update submodules
+      run: |
+        git submodule sync
+        git config --global submodule.lib/modules.update none

```

Comment:
@lalitb  FYI - For some reason I had to exclude manually update of lib/module, otherwise other submodules weren't checked out